tests/test_other.py

```diff
@@ -8031,7 +8031,7 @@ def test_binaryen_metadce_cxx_fastcomp(self, *args):
     # don't compare the # of functions in a main module, which changes a lot
     # TODO(sbc): Investivate why the number of exports is order of magnitude
     # larger for wasm backend.
-    'main_module_1': (['-O3', '-s', 'MAIN_MODULE=1'], 1610, [], [], 517336, 172, 1506, None), # noqa
+    'main_module_1': (['-O3', '-s', 'MAIN_MODULE=1'], 1611, [], [], 517336, 172, 1485, None), # noqa
```

Comment:
It looks like this number also changed recently, so I will probably have to update something locally.